promitor-agent-scraper/README.md

```diff
@@ -103,6 +103,7 @@ their default values.
 | `telemetry.containerLogs.enabled`  | Indication whether or not to send telemetry to container logs | `true`            |
 | `telemetry.containerLogs.logLevel`  | Minimum level of logging for container logs |  |
 | `telemetry.defaultLogLevel`  | Minimum level of logging for all telemetry sinks, unless specified otherwise | `Error`            |
+| `useAzureMonitorSdk`  | Whether to use the Azure Monitor SDK for scraping; alternative is the legacy Azure Management Library | `true`
```

Comment:
We have a `azureMonitor.` section below, maybe we should go with `azureMonitor.useLegacySdk = false`?

Reply:
@hkfgo Can you update this as well and align with other flags please?

Reply:
Done!